screens/Issuers/CredentialTypeSelectionScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import {Modal} from '../../components/ui/Modal';
+import React from 'react';
+import {useIssuerScreenController} from './IssuerScreenController';
+import {FlatList, View} from 'react-native';
+import {CredentialType} from '../../components/openId4VCI/CredentialType';
+import {removeWhiteSpace} from '../../shared/commonUtil';
+import {HomeRouteProps} from '../../routes/routeTypes';
+import {RootRouteProps} from '../../routes';
+import {getDisplayObjectForCurrentLanguage} from '../../shared/openId4VCI/Utils';
+import {Theme} from '../../components/ui/styleUtils';
+import {Column, Text} from '../../components/ui';
+import {useTranslation} from 'react-i18next';
+
+export const CredentialTypeSelectionScreen: React.FC<
+  HomeRouteProps | RootRouteProps
+> = props => {
+  const controller = useIssuerScreenController(props);
+  const {t} = useTranslation('IssuersScreen');
+  const selectedIssuerDisplayObject = getDisplayObjectForCurrentLanguage(
+    controller.selectedIssuer.display,
+  );
+
+  return (
+    <Modal
+      testID="credentialTypeSelectionScreen"
+      isVisible={controller.isSelectingCredentialType}
+      arrowLeft={true}
+      headerTitle={selectedIssuerDisplayObject.name}
+      headerElevation={2}
+      onDismiss={() => controller.CANCEL()}>
+      <Column style={Theme.IssuersScreenStyles.issuerListOuterContainer}>
+        <Text
+          testID="credentialTypeSelectionScreenDescription"
+          style={{
+            ...Theme.TextStyles.regularGrey,
+            ...Theme.IssuersScreenStyles.issuersSearchSubText,
+          }}>
+          {t('credentialTypeDescription')}
+        </Text>
+        <View style={Theme.IssuersScreenStyles.issuersContainer}>
+          <FlatList
+            data={controller.credentialTypes}
+            numColumns={1}
+            keyExtractor={item => item.id}
+            renderItem={({item}) => {
+              return (
+                <CredentialType
+                  testID={removeWhiteSpace(item.id)}
```

Comment:
Id is Issuer id eg: esignet, sunbird?

Reply:
No @swatigoel , Its InsuranceCredential or MosipVerifiableCredential